<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="undoAction"/>
    <addaction name="redoAction"/>
   </widget>
   <widget class="QMenu" name="menu_Tune">
    <property name="title">
     <string>&amp;Image</string>
    </property>
    <addaction name="brightnessAction"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>&amp;Filter</string>
    </property>
    <widget class="QMenu" name="menuSmooth">
     <property name="title">
      <string>&amp;Smooth</string>
     </property>
     <widget class="QMenu" name="menu_Average">
      <property name="title">
       <string>&amp;Average</string>
      </property>
      <addaction name="avg3x3Action"/>
      <addaction name="avg5x5Action"/>
     </widget>
     <widget class="QMenu" name="menu_Median">
      <property name="title">
       <string>&amp;Median</string>
      </property>
      <addaction name="med3x3Action"/>
      <addaction name="med5x5Action"/>
     </widget>
     <addaction name="menu_Average"/>
     <addaction name="menu_Median"/>
     <addaction name="smoothGaussianAction"/>
     <addaction name="smoothBilateralAction"/>
    </widget>
    <addaction name="menuSmooth"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Tune"/>
   <addaction name="menuFilter"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openAction"/>
   <addaction name="saveAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>41</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidget_2">
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
  </widget>
  <action name="openAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open an image file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="toolTip">
    <string>Save image as new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit Darqwin</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="redoAction">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="brightnessAction">
   <property name="text">
    <string>&amp;Brightness</string>
   </property>
   <property name="toolTip">
    <string>Modify image's brightness</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>About DarQwin</string>
   </property>
  </action>
  <action name="smoothGaussianAction">
   <property name="text">
    <string>&amp;Gaussian Blur</string>
   </property>
  </action>
  <action name="smoothBilateralAction">
   <property name="text">
    <string>&amp;Bilateral</string>
   </property>
  </action>
  <action name="avg3x3Action">
   <property name="text">
    <string>3x3</string>
   </property>
  </action>
  <action name="avg5x5Action">
   <property name="text">
    <string>5x5</string>
   </property>
  </action>
  <action name="med3x3Action">
   <property name="text">
    <string>3x3</string>
   </property>
  </action>
  <action name="med5x5Action">
   <property name="text">
    <string>5x5</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
